### PR TITLE
Added in docs that newcomers shouldn't delete forks

### LINKED ISSUE #4387: Specify in the docs that newcomers shouldn't delete forks
Opened via [gitter](https://gitter.im/coala/coala/maintainers/?at=594be7de02c480e6726f92d0) by @meetmangukiya

### PR DESCRIPTION
**Pull request by gureks**
Added a note in the last step.
Closes #4387 

### Checklist

- [X] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [X] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

### CODE REVIEW

**Comment by sidharthskumar** [comment]
needs work
hey @gureks i find [this](http://api.coala.io/en/latest/Developers/Writing_Good_Commits.html) very helpful for making commit message. please check that out :)

File: docs/Developers/Newcomers_Guide.rst [hunk 1]
```diff
@@ -424,6 +424,12 @@ The ``Pull Request`` will automatically update with the newest changes.
 Now you should `tell us about your experience <https://coala.io/newform>`_ and
 go for `a low issue <https://coala.io/low>`__ - they are really rewarding!
 
+
+.. note::
+
+    Do not delete the fork/s subsequent to Pull Request for review or
```
Comment (rhemon) [inline docs/Developers/Newcomers_Guide.rst]: `fork/s` ? shouldn't it be `fork's` ?

Comment (gureks) [inline docs/Developers/Newcomers_Guide.rst]: The issue states to add a note saying not to delete fork after PR. Just made it `fork or forks`.

Comment (Nosferatul) [inline docs/Developers/Newcomers_Guide.rst]: ok, @gureks you can write just `fork` and it's fine

Comment (gureks) [inline docs/Developers/Newcomers_Guide.rst]: Made the changes.

**Comment by Asnelchristian** [comment]
Please modify your commit message to follow the guidelines [here](https://coala.io/commit) : 
Basically: your commit shortlog should be imperative..... 
What about:
```
Newcomer_Guide: Prohibit deleting forks

Pins a note alerting not to delete forks.
Closes #4387
```

**Comment by Asnelchristian** [comment]
And don't forget to rebase ;)

**Comment by Asnelchristian** [comment]
rebase rebase, you forgot to rebase

**Comment by Nosferatul** [comment]
ack 0cefb65

**Comment by Nosferatul** [comment]
@rultor merge

**Pull request merged**